server/API DOCS/GET.md

```diff
@@ -17,5 +17,24 @@
     * **Code:** `401` <br />
       **Message:** `authorization required` <br />
 
+* **Required Filed**
+    * **Auth/ Admin Auth: true/true**
```

Comment:
Here not need to be admin auth so you can write like this.

`Auth/ Admin Auth: true/false `  (First for user Auth/second for admin auth)